types/mongodb/index.d.ts

```diff
@@ -103,54 +153,33 @@ export interface SocketOptions {
     socketTimeoutMS?: number;
 }
 
-//http://mongodb.github.io/node-mongodb-native/2.1/api/Server.html
-export interface ServerOptions {
-    // - specify the number of connections in the pool default:5
-    poolSize?: number;
-    ssl?: boolean;
-    sslValidate?: Object;
-    checkServerIdentity?: boolean | Function;
-    sslCA?: Array<Buffer | string>;
-    sslCert?: Buffer | string;
-    sslKey?: Buffer | string;
-    sslPass?: Buffer | string;
-    socketOptions?: SocketOptions;
+// http://mongodb.github.io/node-mongodb-native/2.2/api/Server.html
+export interface ServerOptions extends SSLOptions, HighAvailabilityOptions{
```

Comment:
I'm not sure if the `ha?: boolean;` from `HighAvailabilityOptions` is present in the `ServerOptions`